recipes/idna-ssl/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "idna_ssl" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "1227e44039bd31e02adaeafdbba61281596d623d222643fb021f87f2144ea147" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```

Comment:
could this be noarch? (just put `python >= 3.5` in the build/run sections)

Reply:
Yes, It did not occur to me that I could limit the python constraint at runtime. It makes perfect sense though.